modules/react/common/stories/Responsive.stories.mdx

```diff
@@ -0,0 +1,314 @@
+import {Meta, ArgsTable} from '@storybook/addon-docs';
+import {Box} from '@workday/canvas-kit-react/layout';
+import {useResizeObserver, useResponsiveContainerStyles} from '@workday/canvas-kit-react/common';
+import {ResponsiveContainer} from './examples/ResponsiveStyles';
+import {ResponsiveViewport} from './examples/ResponsiveViewport';
+
+<Meta title="Hooks and Utilities/Responsive Styling" />
+
+# Responsive Styling
+
+Responsive styling is an approach to web development or a set of best practices, used to create a
+layout that can respond to the device's viewport size.
+
+You can read more about the history of
+[Responsive Design](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design).
+
+## Container and Viewport Based Responsive Styling
+
+We will take a look at Container-based and Viewport-based Responsive Styling, what they are and
+which to use.
```

Comment:
```suggestion
Responsive styling is an approach to web development that adjusts layout and appearance based on the
device's viewport size or an element's container size.

## What is Responsive Styling?

SHORT DESCRIPTION OF RESPONSIVE STYLING GOES HERE.

Read more about the history of
[Responsive Design](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design).
```

Replace the `SHORT DESCRIPTION OF RESPONSIVE STYLING GOES HERE` placeholder with an actual description.

The Canvas Site will strip out the H1 and every paragraph underneath it up until the next H2 (not including the H2). This information is typically rendered in the gray preamble section on the site instead:

<img width="823" alt="image" src="https://user-images.githubusercontent.com/146020/198417018-f11f546d-a238-46c8-9da2-6e8cbe2efabb.png">


If we want the Responsive Design MDN link to show up on the site, it needs to be underneath an H2 following the H1. I've added the `## What is Responsive Styling?` section in my suggestion to give that link a place to live -- it replaces the `## Container and Viewport Based Responsive Styling` which I don't think is needed.